projectgenerator/libqgsprojectgen/dataobjects/layers.py

```diff
@@ -34,6 +34,10 @@ def __init__(self, provider, uri, name, geometry_column=None, wkb_type=QgsWkbTyp
         self.__layer = None
         self.fields = list()
         self.is_domain = is_domain
+        """
+        If is_nmrel is set to true it is a junction table in a N:M relation.
+        Or in ili2db terms, the table is marked as ASSOCIATION in t_ili2db_table_prop.settings.
+        """
         self.is_nmrel = is_nmrel
```

Comment:
Docstrings in python come after the variable. If you find this as unintuitive to read as me, add an empty line before and after

```
self.is_domain = is_domain

self.is_nmrel = is_nmrel
"""
If is_nmrel is set to true it is a junction table in a N:M relation.
Or in ili2db terms, the table is marked as ASSOCIATION in t_ili2db_table_prop.settings.
"""

```